core/templates/dev/head/css/oppia.css

```diff
@@ -1184,7 +1184,7 @@ textarea {
 /* NOTE TO DEVELOPERS: This value should be updated in
    windowDimensionsService if changed.
  */
-@media(max-width: 651px) {
+@media(max-width: 768px) {
```

Comment:
Since this is a max-width media query, having it set to 768px will result in no feedback buttons being visible at width=768px. It would need to be 767px.

Let's just remove this media query entirely, it can be added again if we decide to make the top buttons visible below 768px in the future.